framework/web/CHttpCookie.php

```diff
@@ -48,6 +48,11 @@ class CHttpCookie extends CComponent
 	 * Note, this property is only effective for PHP 5.2.0 or above.
 	 */
 	public $httpOnly=false;
+    /**
+     * @var array Additional setcookie options like "SameSite".
```

Comment:
Formatting seems to be off. Not a huge issue though. Can fix it while merging.

Reply:
Fixed it.